### PR TITLE
Workshop/Speaker/Language/Room/Country changes

### PR DESCRIPTION
**Pull request by zachepilkin**
Views/Controllers added/edited for Workshop/Speaker/Language/Room/Country changes entities

### CODE REVIEW

File: AzureDay.WebApp/AzureDay.WebApp.Database/Services/CountryService.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace AzureDay.WebApp.Database.Services
+{
+    public sealed class CountryService : BaseService<CountryEntity, int>
+    {
+        protected override List<CountryEntity> PopulateStorage()
+        {
+            return new List<CountryEntity>
+            {
+                new CountryEntity {Id = 1, ImageUrl = "https://azureday2017ua.blob.core.windows.net/images/flags/ua.png", Title = Localization.App.Service.Country.Ukraine}
```
Comment (boyko-ant) [inline AzureDay.WebApp/AzureDay.WebApp.Database/Services/CountryService.cs]: please add this image to source code repo, we should not use external resources except database (table storage)

File: AzureDay.WebApp/AzureDay.WebApp.Database/Services/SpeakerService.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        public SpeakerEntity ABoyko()
+        {
+            return new SpeakerEntity
+            {
+                Id = "ABoyko",
+                FirstName = Localization.App.Service.Speaker.ABoyko.FirstName,
+                LastName = Localization.App.Service.Speaker.ABoyko.LastName,
+                Bio = Localization.App.Service.Speaker.ABoyko.Bio.Replace(Environment.NewLine, "<br/>"),
+                Country = _countryService.Ukraine,
+                PhotoUrl = "https://azureday2017ua.blob.core.windows.net/images/avatars/ABoyko.jpg",
```
Comment (boyko-ant) [inline AzureDay.WebApp/AzureDay.WebApp.Database/Services/SpeakerService.cs]: please add this image to source code repo, we should not use external resources except database (table storage)

File: AzureDay.WebApp/AzureDay.WebApp.Database/Services/SpeakerService.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        public SpeakerEntity DIvanov()
+        {
+            return new SpeakerEntity
+            {
+                Id = "DIvanov",
+                FirstName = Localization.App.Service.Speaker.DIvanov.FirstName,
+                LastName = Localization.App.Service.Speaker.DIvanov.LastName,
+                Bio = Localization.App.Service.Speaker.DIvanov.Bio.Replace(Environment.NewLine, "<br/>"),
+                Country = _countryService.Ukraine,
+                PhotoUrl = "https://azureday2017ua.blob.core.windows.net/images/avatars/DIvanov.jpg",
```
Comment (boyko-ant) [inline AzureDay.WebApp/AzureDay.WebApp.Database/Services/SpeakerService.cs]: please add this image to source code repo, we should not use external resources except database (table storage)

**Comment by zachepilkin** [comment]
local resources for images added

**Pull request merged**